openapi-specs/locations-inside-prison-api.json

```diff
@@ -0,0 +1 @@
"type":"array","description":"Specialist Cell Types","items":{"type":"string","description":"Specialist Cell Types","enum":["BIOHAZARD_DIRTY_PROTEST","CAT_A","CONSTANT_SUPERVISION","CSU","DRY","ESCAPE_LIST","FIRE_RESISTANT","FIXES_FURNITURE","ISOLATION_DISEASES","LIGATURE_RESISTANT","LISTENER_CRISIS","LOW_MOBILITY","MEDICAL","MOTHER_AND_BABY","SOUND_RESISTANT","UNFURNISHED","WHEELCHAIR_ACCESSIBLE"]}},"isCell":{"type":"boolean"},"isDeactivated":{"type":"boolean"}},"description":"Request to update a location"},"GetDlqResult":{"required":["messages","messagesFoundCount","messagesReturnedCount"],"type":"object","properties":{"messagesFoundCount":{"type":"integer","format":"int32"},"messagesReturnedCount":{"type":"integer","format":"int32"},"messages":{"type":"array","items":{"$ref":"#/components/schemas/DlqMessage"}}}},"PageLocation":{"type":"object","properties":{"totalPages":{"type":"integer","format":"int32"},"totalElements":{"type":"integer","format":"int64"},"first":{"type":"boolean"},"last":{"type":"boolean"},"size":{"type":"integer","format":"int32"},"content":{"type":"array","items":{"$ref":"#/components/schemas/Location"}},"number":{"type":"integer","format":"int32"},"sort":{"type":"array","items":{"$ref":"#/components/schemas/SortObject"}},"numberOfElements":{"type":"integer","format":"int32"},"pageable":{"$ref":"#/components/schemas/PageableObject"},"empty":{"type":"boolean"}}},"PageableObject":{"type":"object","properties":{"offset":{"type":"integer","format":"int64"},"sort":{"type":"array","items":{"$ref":"#/components/schemas/SortObject"}},"pageSize":{"type":"integer","format":"int32"},"pageNumber":{"type":"integer","format":"int32"},"unpaged":{"type":"boolean"},"paged":{"type":"boolean"}}},"SortObject":{"type":"object","properties":{"direction":{"type":"string"},"nullHandling":{"type":"string"},"ascending":{"type":"boolean"},"property":{"type":"string"},"ignoreCase":{"type":"boolean"}}},"Constant":{"required":["description","key"],"type":"object","properties":{"key":{"type":"string"},"description":{"type":"string"}}},"CompoundConstant":{"required":["description","key","values"],"type":"object","properties":{"key":{"type":"string"},"description":{"type":"string"},"values":{"type":"array","items":{"$ref":"#/components/schemas/Constant"}}}}},"securitySchemes":{"bearer-jwt":{"type":"http","name":"Authorization","in":"header","scheme":"bearer","bearerFormat":"JWT"},"hmpps-auth":{"type":"oauth2","flows":{"clientCredentials":{"tokenUrl":"https://sign-in-dev.hmpps.service.justice.gov.uk/auth/oauth/token","scopes":{"read":"Allows read of data","write":"Allows write of data"}},"authorizationCode":{"authorizationUrl":"https://sign-in-dev.hmpps.service.justice.gov.uk/auth/oauth/authorize","tokenUrl":"https://sign-in-dev.hmpps.service.justice.gov.uk/auth/oauth/token","scopes":{"read":"Allows read of data","write":"Allows write of data"}}},"openIdConnectUrl":"https://sign-in-dev.hmpps.service.justice.gov.uk/auth/.well-known/openid-configuration"}}}}
```

Comment:
Do we normally check in these generated files? Couldn't remember.

Reply:
We do yes.  The other option is to point at an actual URL e.g. the DEV URL.  One of the risks being if we do this things change and breaks our service without us knowing. Essentially it's a trade off.